frontend/src/config/mozambique/layers.json

```diff
@@ -2954,5 +2954,74 @@
     "start_date": "2020-12-01",
     "end_date": "2021-07-31",
     "opacity": 0.7
+  },
+  "google_flood_status_at_gauges": {
+    "title": "Current Flood Status (Google AI)",
+    "type": "point_data",
+    "hex_display": false,
+    "data": "http://0.0.0.0:80/google-floods/gauges/?region_code=MZ",
+    "data_field": "severity",
+    "data_field_type": "text",
+    "opacity": 0.9,
+    "legend_text": "Current flood status at verified gauges. Visit [Google Research](https://sites.research.google/floodforecasting/) about Google's AI Forecasting models.",
+    "legend": [
+      { "label": "Extreme", "value": "EXTREME", "color": "#a70606" },
+      { "label": "Danger", "value": "SEVERE", "color": "#ea250a" },
+      { "label": "Warning", "value": "ABOVE_NORMAL", "color": "#fba705" },
+      { "label": "Normal", "value": "NO_FLOODING", "color": "#089180" },
+      { "label": "Unknown", "value": "UNKNOWN", "color": "#858585" }
+    ],
+    "feature_info_title": {
+      "siteName": {
+        "type": "text",
+        "template": "Site: %s",
+        "visibility": "if-defined"
+      },
+      "river": {
+        "type": "text",
+        "template": "River: %s",
+        "visibility": "if-defined"
+      },
+      "gaugeId": {
+        "type": "text",
+        "template": "Gauge ID: %s",
```

Comment:
Let's think of a way to make this template translatable?

Reply:
We haven't really done something like that in the past, we would usually have the template be a title and then handle them separately. So instead of template you would just have `title: Gauge ID`.

Another option is to use the translation mapping and have the template be: `Gauge ID: {id}`, then 

          <Trans values={{ id }}>
            <Typography>{t(template)}</Typography>
          </Trans>

Reply:
Thanks @ericboucher! I'm leaning towards the latter option (translation mapping). We used these templates in react-i18next pretty extensively in the distant past (my Hilton Hotels days). Once we started using them, it gave us a ton of flexibility in terms of copy. If you're ok with it, I'll implement it this way.

Reply:
Updated